static/js/containers/ProfilePage_test.js

```diff
@@ -40,6 +40,7 @@ describe("ProfilePage", function() {
     listenForActions = helper.listenForActions.bind(helper);
     renderComponent = helper.renderComponent.bind(helper);
     patchUserProfileStub = helper.sandbox.stub(api, 'patchUserProfile');
+    HTMLDivElement.prototype.scrollIntoView = helper.sandbox.stub();
```

Comment:
It might make sense to set this up in `integration_test_helper.js` so that it's there for all of our tests. What do you think?